WSL/toc.yml

```diff
@@ -26,6 +26,8 @@
       href: tutorials/wsl-database.md
     - name: Get started with Docker remote containers
       href: tutorials/wsl-containers.md
+    - name: Get started with Visual Studio for C++ development
+      href: /cpp-docs/build/walkthrough-build-debug-wsl2.md
```

Comment:
In the preview the link to this doc is giving me a 404 error. 
@mattwojo would you know how to properly link this?